src/main/resources/META-INF/primefaces-p.taglib.xml

```diff
@@ -23902,6 +23902,22 @@
             <name>decimalPlaces</name>
             <required>false</required>
             <type>java.lang.String</type>
+        </attribute>
+		<attribute>
+            <description>
+                <![CDATA[Decimal separator char. Default is taken from the view's locale.]]>
```

Comment:
`Character` or `String`? `DecimalFormatSymbols` uses `Character`, so this should be sufficient for us as well?

Reply:
Character is enough for me. However I think that string is more flexible (for example someone may need to use space + separator) and it is consistent with p:inputNumber.